data/mods/Magiclysm/items/ethereal_items.json

```
@@ -757,5 +757,58 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_PURE", "add": 10 } ] } ]
     }
+  },
+  {
+    "id": "azoth_sword",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": { "str": "Azoth" },
+    "description": "Clear energy blade, powered by users mana and spitit.",
+    "material": [ "concentrated_mana" ],
+    "flags": [ "UNBREAKABLE_MELEE", "NONCONDUCTIVE", "NO_REPAIR", "NO_SALVAGE", "MAGIC_FOCUS", "LIGHT_10", "TRADER_AVOID" ],
+    "techniques": [ "WBLOCK_1", "WIDE", "SPIN" ],
+    "weight": "200 g",
+    "volume": "1250 ml",
+    "longest_side": "100 cm",
+    "bashing": 15,
+    "cutting": 45,
+    "to_hit": 2
+  },
+  {
+    "id": "wigar_armor",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "wigar armor" },
+    "description": "Clear energy armor, filled with unquenchable spirit and arcane energy.",
+    "weight": "400 g",
+    "volume": "25 L",
+    "material": [ "concentrated_mana" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "covers": [ "head", "mouth", "eyes", "torso", "arm_l", "arm_r", "hand_l", "hand_r", "leg_l", "leg_r", "foot_l", "foot_r" ],
+    "coverage": 100,
+    "encumbrance": 15,
+    "material_thickness": 1.7,
+    "environmental_protection": 5,
+    "flags": [ "ONLY_ONE", "OVERSIZE", "PERSONAL", "STURDY" ]
+  },
+  {
+    "id": "aura_auroral_guard",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "auroral shell" },
+    "description": "Sturdy shell what wrap your body and gain almost perfect defence.",
```

```suggestion
    "description": "A sturdy shell that wraps your body and gives an almost perfect defense.",
```
I'm not sure if I would quite say "almost perfect"; mithril appears to be better in some regards, and 15 encumbrance - particularly on the mouth and eyes! - is not inconsiderable.